tensorflow/stream_executor/rocm/rocm_activation.cc

```diff
@@ -0,0 +1,48 @@
+/* Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include "tensorflow/stream_executor/gpu/gpu_activation.h"
+
+#include "tensorflow/stream_executor/gpu/gpu_driver.h"
+#include "tensorflow/stream_executor/stream_executor.h"
+#include "tensorflow/stream_executor/stream_executor_internal.h"
+
+namespace stream_executor {
+namespace gpu {
+
+GpuContext* ExtractGpuContext(GpuExecutor* rocm_exec);
```

Comment:
This file and cuda_activation.cc are identical except for variable names. Are they going to diverge in the future? Maybe just merge them?

Reply:
will merge `rocm/rocm_activation.cc` and `cuda/cuda_activation.cc` into a common `gpu/gpu_activation.cc` in the next commit.

I did not realize that you were okay with merging the .cc files as well (as opposed to just the header files)